modular_exponential/ModularExponential.java

```diff
@@ -0,0 +1,20 @@
+public class ModularExponential {
+	public static int modularExponential(int base, int power, int mod) {
+		int result = 1;
```

Comment:
Only for result should do.
But use a larger data type (long?) for the parameters.